### DISCUSSION TITLE
Automatic Dependency Updates for JBang scripts

### DISCUSSION THREAD

**Discussion by maxandersen** [jbangdev/jbang#1661] [General]
This is amazing. 

@helpermethod tipped me that renovate can be taught how to detect //DEPS and make PR's to update out-dated dependencies.

I enhanced his approach and you can now do: 

`jbang init -t renovate@jbangdev .github/renovate.json`

And get automatic dependency checking on github in your jbang scripts.

Below is example for me applying this to jbang's own jbang-catalog:

<img width="1496" alt="2023-08-03_09-47-11" src="https://github.com/jbangdev/jbang/assets/54129/7e03cf19-2d37-4a1d-9930-f92818d8e14c">

How cool is that?!

**Comment by quintesse** [comment]
Nice! Btw, I think we need to start the matches with a `^` to be sure. I for example often put an extra slash in front of tags I want to (temporarily) disable, eg: `///DEPS unwanted:artifact:1.2.3`

**Reply by maxandersen** [reply]
Good point. PR welcome :)
